drivers/hmis/app/views/hmis_client/services/_table.haml

```diff
@@ -0,0 +1,27 @@
+.table-responsive
+  %table.table
+    %thead
+      %tr
+        %th
+        %th Project
+        %th Entry Date
+        %th Service Date Provided
+        %th Service Name
+        %th
+    %tbody
+      - custom_services.each do |service|
+        %tr
+          %td
+            .ds{class: "ds-color-#{service.data_source_id}"}= ds_short_name_for(service.client.id)
+          %td= service.project.name
+          %td= service.enrollment.entry_date
+          %td= service.date_provided
+          %td= service.service_type.custom_service_category.name == service.service_name ? service.service_name : "#{service.service_type.custom_service_category.name} - #{service.service_name}"
```

Comment:
This unwieldy line is meant to match the HMIS frontend's behavior [here](https://github.com/greenriver/hmis-frontend/blob/325fa54119f0346a9a520318303acda99eaf82e5/src/modules/enrollment/components/dashboardPages/EnrollmentServicesPage.tsx#L30-L32), but I wanted to check and confirm that it's a good idea/worth the complexity?

Reply:
Can that be added as a method on `service`?  I'm a big fan of fat models, thin views.

Reply:
Thanks, that's much better! 😄